src/cli/util.js

```diff
@@ -433,7 +437,31 @@ function formatFiles(context) {
     context.logger.log("Checking formatting...");
   }
 
-  for (const pathOrError of expandPatterns(context)) {
+  const willWriteOutput =
+    !context.argv.check &&
+    !context.argv["list-different"] &&
+    !context.argv["debug-check"];
+  let files;
+  if (willWriteOutput) {
+    if (context.filePatterns.length !== 1) {
+      context.logger.error(
+        `Expect only one filepath when format to stdout, ${context.filePatterns.length} given.`
```

Comment:
If you pass _one_ pattern (a glob) that matches multiple files, will this say “expected 1 filepath, 1 given”?

Reply:
No, glob is not supported at all.